.github/workflows/cibuild.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install
       run: |
+        sudo apt-get install bwa samtools fastqc
+        curl -L -O http://ccb.jhu.edu/software/FLASH/FLASH-1.2.11-Linux-x86_64.tar.gz && tar -xzf FLASH-1.2.11-Linux-x86_64.tar.gz && sudo cp FLASH-1.2.11-Linux-x86_64/flash /usr/local/bin/ && rm FLASH-1.2.11*
```

Comment:
BWA, SAMtools, and FastQC can be installed using Ubuntu's package manager. FLASH cannot.